Words replacements (#918)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc

     id: eclipse/che-machine-exec-plugin/0.0.1
----

> Both types above using id, which is slash-separated publisher, name and version of plug-in from Che Plugin registry. +

Reviewer: ```suggestion
Both types above use an ID, which is slash-separated publisher, name and version of plug-in from the Che Plug-in registry. +
```
